lightningd/notification.c

```diff
@@ -575,7 +599,7 @@ void notify_balance_snapshot(struct lightningd *ld,
 static void block_added_notification_serialize(struct json_stream *stream,
 					       struct block *block)
 {
-	json_object_start(stream, "block");
+	json_object_start(stream, "block_added");
```

Comment:
I see your motivation here, but I think you need to do *both* for compatibility?

Reply:
Sorry, forgot to add deprecated_apis logic. Updated the code to keep it compatible with commit 769e59f472402b1892699bc981acc382aa198a8d